packages/dropdown/src/react/__stories__/index.story.tsx

```diff
@@ -1,834 +1,285 @@
-          </>
-        }
-      />
+      <Dropdown.Item onClick={action('click item 1')} value={1}>
+        Item 1
+      </Dropdown.Item>
+      <Dropdown.Item onClick={action('click item 2')} value={2}>
+        Item 2
+      </Dropdown.Item>
+      <Dropdown.Item onClick={action('click item 3')} value={3}>
+        Item 3
+      </Dropdown.Item>
     </>
-  ))
-  .add('shows pre-selected item by label', () => (
-    <Dropdown
-      placeholder="Shown because can't find option"
-      value="Preselected three item"
-      menu={
-        <>
-          <Dropdown.Item>One item</Dropdown.Item>
-          <Dropdown.Item>Two item</Dropdown.Item>
-          <Dropdown.Item>Preselected three item</Dropdown.Item>
-        </>
-      }
-    />
-  ))
-  .add('shows pre-selected item by value', () => (
-    <Dropdown
-      placeholder="Not shown until empty"
-      value="h"
-      menu={
-        <>
-          <Dropdown.Item value="o">One item</Dropdown.Item>
-          <Dropdown.Item value="w">Two item</Dropdown.Item>
-          <Dropdown.Item value="h">Preselected three item</Dropdown.Item>
-        </>
-      }
-    />
-  ))
-  .add('value is not an item in the list', () => (
-    <Dropdown
-      placeholder="Shown because can't find option"
-      value="unknown"
-      menu={
-        <>
-          <Dropdown.Item value="o">One item</Dropdown.Item>
-          <Dropdown.Item value="w">Two item</Dropdown.Item>
-          <Dropdown.Item value="h">Preselected three item</Dropdown.Item>
-        </>
-      }
-    />
-  ))
-  .add('external state change', () => {
-    function StateChange() {
-      const values = ['o', 'w', 'h']
-      const [value, setValue] = useState<ReactText | undefined>(values[0])
-      useEffect(() => {
-        const timerId = setInterval(() => {
-          const i = values.findIndex(v => v === value)
-          setValue(i < values.length - 1 ? values[i + 1] : values[0])
-        }, 1000)
-
-        return () => clearInterval(timerId)
-      })
-
-      return (
-        <Dropdown
-          placeholder="Not shown until empty"
-          value={value}
-          menu={
-            <>
-              <Dropdown.Item value="o">One item</Dropdown.Item>
-              <Dropdown.Item value="w">Two item</Dropdown.Item>
-              <Dropdown.Item value="h">Preselected three item</Dropdown.Item>
-            </>
-          }
-        />
-      )
-    }
-    return <StateChange />
-  })
+  ),
+  placeholder: 'Some placeholder',
+  subLabel: 'The sub label'
+}
 
-  .add('test the spec', () => (
-    <Dropdown
-      placeholder="Select"
-      value="w"
-      menu={[
-        <Dropdown.Item key="1" value="o">
-          One
-        </Dropdown.Item>,
-        <Dropdown.Item key="2" value="w">
-          Two
-        </Dropdown.Item>,
-        <Dropdown.Item key="3" value="h">
-          Three
-        </Dropdown.Item>
-      ]}
-    />
-  ))
+export default {
+  title: 'Components/Dropdown',
```

Comment:
for the stories: i noticed some overlap on what was being validated. I attempted to simplify and also broke the file up into 3